BiweeklyMeetings/2022-02-15.md

```diff
@@ -0,0 +1,382 @@
+
+**Matthias** - So if the intent is actually to show that this was a path that was discussed, but sort of rejected, there is always the option of merging it as a rejected CIP or obsolete CIP. So that it can really show that, yeah, this was discussed and this was really rejected for these and those reasons.
+
+**Maria** - Could you make a quick summary? I think you said there's three collateral PRs, but I've only been able to locate two of them..
+
+**Matthias** - I think the first, or third one should be, or will be closed anyway by Sebastian. Yeah. It's the PR 104, which is now labeled as likely deprecated, and then, so Sebastian, since you also the author of that one, I guess you can move forward and close it.
+
+**Sebastien** - This is from before the hard fork, before we discussed collateral output and collateral output
+**Matthias** - I remember that. There was also the case of merging that one as a rejected obsolete CIP. Just to keep the discussion recorded.
+**Maria** - Okay, so we have this collateral key that is going to be closed or marked as obsolete?
+Then we have a collateral reward that is going to be closed, but we have time to do that?
+And then there's the collateral output, that it needs to be done before the hard fork, so there's no rush, but ... Shall we put them on review?
+
+**Matthias** - We could probably put that on review for next week or next month, at least, to also give you a bit of time. It seems like the spec is being worked on by the ledger team in parallel, so that already gives a more streamlined path to implementation, possibly. It is quite observable from past to be active. I'm not sure if they will be done in two weeks from now, but yeah, maybe we can have it as review for the next meeting, and if it's still being cooked, we can look at it back. Yeah, more weeks for the next one. I think we'll just stop it here, and thank you everyone for joining, coming, and commenting. It's good actually, to see more and more people joining these meetings, and see you in two weeks.
+
+=> [CIP-1856 | Collateral Key derivation](https://github.com/cardano-foundation/CIPs/pull/104) moved to Triage for next meeting
+=> [Collateral reward #217](https://github.com/cardano-foundation/CIPs/pull/217) moved to Triage for next meeting
+=> [Collateral output #216](https://github.com/cardano-foundation/CIPs/pull/216) moved to review
+
+
+
+
+### Close    
+
+=> Merged minor changes to CIP-0021 - [PR157 - Adding multisig changes to CIP-0021](https://github.com/cardano-foundation/CIPs/pull/157)    
```

Comment:
All these lines are being folded together into a single unreadable paragraph.  Could be easily made into a bulleted list, or blank lines added in between items, like here: https://github.com/mangelsjover/CIPs/blob/master/BiweeklyMeetings/2022-03-01.md

Reply:
Thanks, I did that Close items on the link as well, just sent it to Maria for upload. I would make the same changes for this notes, when I have time.